samples/interceptors/wso2-micro-integrator/README.md

```diff
@@ -0,0 +1,69 @@
+# Sample Interceptor - WSO2 Micro Integrator
+
+The interceptor service is generated using the interceptor service Open API definition.
+[Swagger Editor](https://editor.swagger.io/) is used to generate the Synapse API configuration file in Micro Integrator.
```

Comment:
Did we use [Swagger Editor](https://editor.swagger.io/) to generate synapse configs?

Reply:
No, I have corrected the sentence